src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -0,0 +1,584 @@
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.NONE}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.NONE</div>
+                </Box>
+            </div>
+            <div className="monday-storybook-box-padding-wrapper">
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XS}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.XS</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.SMALL}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.SMALL</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.MEDIUM}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.MEDIUM</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.LARGE}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.LARGE</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XL}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.XL</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XXL}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.XXL</div>
+                </Box>
+            </div>
+            <div className={"monday-storybook-box-padding-wrapper"}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XXXL}
+                >
+                    <div className="monday-storybook-box-padding-inner">Box.paddings.XXXL</div>
+                </Box>
+            </div>
+        </div>
+    </Story>
+</Canvas>
+
+#### Property variations per each size:
+```
+padding
+paddingX (x axis: left, right)
+paddingY (y axis: top, bottom)
+paddingTop
+paddingEnd
+paddingBottom
+paddingStart
+```
+
+### Opacity
+<Canvas>
```

Comment:
maybe change api to be boolean prop: disabled will e more similar to other components? 
if we will have multiple opacity states in the feature it will be a scale and not with meaning (like spacing or border)

Reply:
done.